perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ResourcesManagerMethod.java

```diff
@@ -1419,5 +1419,55 @@ public Object call(ApiCaller ac, Deserializer parms) throws PerunException {
 			return ac.getResourcesManager().getGroupAssignments(ac.getSession(),
 				ac.getResourceById(parms.readInt("resource")), attrNames);
 		}
+	},
+
+	/*#
+	 * Try to activate the group-resource status. If the async is set to false, the validation is performed
+	 * synchronously. The assignment will be either ACTIVE, in case of a successful synchronous call, or it will be
+	 * PROCESSING in case of an asynchronous call. After the async validation, the state can be either ACTIVE or
+	 * FAILED.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param resource int Resource <code>id</code>
+	 * @param async boolean if true the validation is performed asynchronously
+	 * @throw GroupNotExistsException when the group doesn't exist
+	 * @throw ResourceNotExistsException when the resource doesn't exist
+	 * @throw WrongAttributeValueException when an attribute value has wrong/illegal syntax
+	 * @throw WrongReferenceAttributeValueException when an attribute value has wrong/illegal semantics
+	 * @throw GroupResourceMismatchException when the given group and resource are not from the same VO
+	 * @throw GroupNotDefinedOnResourceException when the group-resource assignment doesn't exist
+	 */
+	activateGroupResourceAssignment {
+		@Override
+		public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
+			ac.getResourcesManager().activateGroupResourceAssignment(ac.getSession(),
```

Comment:
Please, force here the async parameter to be always false, or throw some exception.

Reply:
Also, mention in the javadoc that this method shouldn't be used, yet.